homeassistant/components/yi/camera.py

```diff
@@ -2,21 +2,24 @@
 import asyncio
 import logging
 
+from aioftp import Client, StatusCodeError  # pylint: disable=import-error
```

Comment:
Why disable pylint here?

Reply:
A misunderstanding, `haffmpeg` needs it, but not `aioftp`, corrected

Reply:
Why does `haffmpeg` need it?

Reply:
When I launch pylint locally it fails because of `haffmpeg` (screenshot in 3 hours)

Reply:
Understood. We shouldn’t include `pylint: disable` directives in the production codebase simply because they don’t work locally. They ending up masking potential issues in an environment where we expect `haffmpeg` to be present.

Two options:

1. Use a Python virtualenv while developing and install needed dependencies there. Then, run `pylint` with that virtualenv activates.
2. If you want a quick-and-dirty method to still check for linting errors, you can leave these directives in place, run `pylint`, and once you confirm, remove these directives.

Reply:
@bachya 
Always developing in the `venv` and installing deps : 
<img width="691" alt="Capture d’écran 2019-11-25 à 22 56 40" src="https://user-images.githubusercontent.com/14821482/69581542-fd946400-0fd6-11ea-96fd-c31266b575c9.png">

But `haffmpeg` can't be installed ... it's not in the [manifest.json](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/yi/manifest.json), maybe the solution will be to add it there ?

But yeah I completely agree with you, this should be in production, removing and see what the pipe will say.

Reply:
Great! Thank you. FYI, you install it this way:

```
pip install ha-ffmpeg
```

It will be installed on-demand when an integration requires it.